src/index.html

```diff
@@ -1,12 +1,403 @@
+
+          <li class="nav__item">
+            <a href="#Tablets" class="nav__link">
+              Tablets
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#Photo" class="nav__link">
+              Photo
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#Video" class="nav__link">
+              Video
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <main>
+      <div class="catalog">
+        <div class="card catalog__card" data-qa="card">
+          <img class="card__picture" src="./images/imac.jpeg"
+            alt="apple a1419 imac 27  5k">
+
+          <h2 class="card__main-text">
+            APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+          </h2>
+
+          <h2 class="card__product-code">
+            Product code: 195434
+          </h2>
+
+          <div class="card__star-reviews">
+            <ul class="star star--4">
+              <li class="star__stars"></li>
+              <li class="star__stars"></li>
+              <li class="star__stars"></li>
+              <li class="star__stars"></li>
+              <li class="star__stars"></li>
+            </ul>
+
+            <div class="card__reviews">
+              Reviews: 5
+            </div>
+          </div>
+
+          <div class="card__price">
+            <p class="card__price-text">Price:</p>
+            <p class="card__price-value">$2,199</p>
+          </div>
+
+          <a 
+            href="#buy" 
+            class="card__button"
+            data-qa="card-hover"
+          >
+            buy
+          </a>
+        </div>
+        <div class="card catalog__card">
+          <img 
+            class="card__picture" 
+            src="./images/imac.jpeg"
+            alt="apple a1419 imac 27  5k"
+          >
+
+          <h2 class="card__main-text">
+            APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+          </h2>
+
+          <h2 class="card__product-code">
+            Product code: 195434
+          </h2>
+
+          <div class="card__star-reviews">
+            <ul class="star star--4">
+              <li class="star__stars"></li>
+              <li class="star__stars"></li>
+              <li class="star__stars"></li>
+              <li class="star__stars"></li>
+              <li class="star__stars"></li>
+            </ul>
+
+            <div class="card__reviews">
+              Reviews: 5
+            </div>
+          </div>
+
+          <div class="card__price">
+            <p class="card__price-text">Price:</p>
+            <p class="card__price-value">$2,199</p>
+          </div>
+
+          <a 
+            href="#buy" 
+            class="card__button" 
+            data-qa="hover"
+          >
+            buy
+          </a>
+        </div>
+        <div class="card catalog__card">
+          <img class="card__picture" src="./images/imac.jpeg"
+            alt="apple a1419 imac 27  5k">
```

Comment:
Improve code style. Check yourself with checklist and fix all cases if exist
```suggestion
          <img
            class="card__picture"
            src="./images/imac.jpeg"
            alt="apple a1419 imac 27  5k
          >
```